angular_components/CHANGELOG.md

```diff
@@ -1,9 +1,90 @@
+## 0.11.0
+
+### New Component
+* Simple HTML Component.
+
+### Component Updates
+
+#### Material Chips
+* Add a focused style to delete icon.
+* Remove default value from `$max-chip-width` in the `material-chip-max-width`
+  Sass mixin.
+* Remove `material-chips-margin` Sass mixin.
+
+#### Material Date Range Picker
+* Make dropdown not tabbable so it can be opened using the top level button
+  decorator.
+* Add NextNDaysFromToday class.
+* Auto scroll to the bottom when comparison is turned on by the user.
+* Fix constrained sizing on Firefox browser.
+* Make intl messages `final`.
+
+#### Material Dialog
+* Fix size of full screen dialog.
+* Close parent modal on escape key by default.
+
+#### Material Input
+* Update the integer error message per Editorial feedback.
+* Add Sass mixin for changing the color of the counter.
+* Fix a11y fix when there is a `labeledby` id and a aria label specified.
+
+#### Material Menu
+* Add a drop-in replacement for `secondaryIcon` - `itemSuffix` that removes the
+  boilerplate of creating an observable list for a single element.
+* Automatically expand a collapsed category when keyboard navigating to a child
+  item.
+* Remove usages of secondary icon and related fields/methods from menu item
+  model.
+* Prevent refocus on a menu item when the menu is closing.
+
+#### Material Popup
+* Correctly restore focus in nested popups.
+
+#### Material Select
+* Add type parameters on `BaseDropdownSelectValueAccessor` and subclasses.
+* Set max-width to 100% on `dynamic-item`.
+* Removing Sass mixin for setting the width of the container element in
+  dropdowns with factoryRenderers to 100%, as this was adopted as the standard.
+* Use the generic type parameter from MaterialDropdownSelectComponent on the
+  ActivateItemOnKeyPressMixin.
```

Comment:
nits: put MaterialDropdownSelectComponent and ActivateItemOnKeyPressMixin class names in code font